tests/jax/galsim/test_image_jax.py

```diff
@@ -4500,57 +4502,59 @@ def test_wrap():
         im_wrap.bounds, b, "image.wrap(%s) does not have the correct bounds"
     )
 
-    im2_wrap = im2.wrap(b2, hermitian="y")
-    # print('im_test = ',im_test[b2].array)
-    # print('im2_wrap = ',im2_wrap.array)
-    # print('diff = ',im2_wrap.array-im_test[b2].array)
-    np.testing.assert_array_almost_equal(
-        im2_wrap.array,
-        im_test[b2].array,
-        12,
-        "image.wrap(%s) did not match expectation" % b,
-    )
-    np.testing.assert_array_equal(
-        im2_wrap.array,
-        im2[b2].array,
-        "image.wrap(%s) did not return the right subimage",
-    )
-    np.testing.assert_equal(
-        im2_wrap.bounds, b2, "image.wrap(%s) does not have the correct bounds"
-    )
-
-    im3_wrap = im3.wrap(b3, hermitian="x")
-    # print('im_test = ',im_test[b3].array)
-    # print('im3_wrap = ',im3_wrap.array)
-    # print('diff = ',im3_wrap.array-im_test[b3].array)
-    np.testing.assert_array_almost_equal(
-        im3_wrap.array,
-        im_test[b3].array,
-        12,
-        "image.wrap(%s) did not match expectation" % b,
-    )
-    np.testing.assert_array_equal(
-        im3_wrap.array,
-        im3[b3].array,
-        "image.wrap(%s) did not return the right subimage",
-    )
-    np.testing.assert_equal(
-        im3_wrap.bounds, b3, "image.wrap(%s) does not have the correct bounds"
-    )
+    # FIXME: turn on when hermitian wrapping is implemented
```

Comment:
Might you want to document the remaining issue with hermitian images in an issue in addition to this FIXME?

Reply:
also if you might mention when this comes up that would be great, I'm not too familiar with when we might need to do wrapping of hermitian images

Reply:
I added a note to the contributing document.